python/pyspark/ml/feature.py

```diff
@@ -1270,11 +1319,11 @@ class Imputer(JavaEstimator, _ImputerParams, JavaMLReadable, JavaMLWritable):
     """
 
     @keyword_only
-    def __init__(self, strategy="mean", missingValue=float("nan"), inputCols=None,
-                 outputCols=None):
+    def __init__(self, strategy="mean", missingValue=float("nan"), inputCol=None, inputCols=None,
```

Comment:
Here we must place newly added params `inputCol` & `outputCol` behind existing ones